types/src/test/scala/roc/types/decoders/NumericDecodersSpec.scala

```diff
@@ -14,36 +14,26 @@ final class NumericDecodersSpec extends Specification with ScalaCheck { def is =
   Short Decoder
     must return the correct Short when Text Decoding a valid Short String                 ${ShortDecoder().testValidTextDecoding}
     must throw a ElementDecodingFailure when TextDecoding an Invalid Short String         ${ShortDecoder().testInvalidTextDecoding}
-    must return the correct Short when Binary Decoding a valid Short Byte Array           ${ShortDecoder().testValidBinaryDecoding}
```

Comment:
All the binary tests were moved to integration tests - that way, they can test against Postgres itself.  Check out the new tests to see what I mean.